lib/rgeo/coord_sys/crs_to_crs.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "singleton"
+module RGeo
+  module CoordSys
+    # This is a Ruby wrapper around a proj crs_to_crs
+    # A crs_to_crs transformation object that is a pipeline between two known coordinate reference systems.
+    # https://proj.org/development/reference/functions.html#c.proj_create_crs_to_crs
+    class CRSToCRS
+      def transform_coords(x, y, z)
+        _transform_coords(x, y, z)
```

Comment:
I'm not sure why we need this?

It might be about documenting the code however there is no method documentation here and you could do it in C: https://ruby.github.io/rdoc/RDoc/Parser/C.html

Reply:
Technically you are right, there is no need to redefine it.

But I followed the convention that the C methods should be very private and prefixed by "_" and have the public method `transform_coords` declared in Ruby, so that's it's easier to find.
Agreed I should add documentation and will do.